modules/rosa-basic-concepts.adoc

```diff
@@ -0,0 +1,52 @@
+
+Pod:: Every containerized app that is deployed into a cluster is deployed, run, and managed by a Kubernetes resource that is called a pod. Pods represent small deployable units in a Kubernetes cluster and are used to group the containers that must be treated as a single unit. In most cases, each container is deployed in its own pod. However, an app can require a container and other helper containers to be deployed into one pod so that those containers can be addressed by using the same private IP address.
+
+App:: An app can refer to a complete app or a component of an app. You can deploy components of an app in separate pods or separate compute nodes.
+
+[id="rosa-containers_{context}"]
+== Containers
+
+Containers provide a standard way to package your application's code, configurations, and dependencies into a single unit that can run as a resource-isolated process on a compute server. To run your app in Kubernetes on {product-title} ROSA, you must first containerize your app by creating a container image that you store in a container registry. For complete information about Docker, see the link:https://docs.docker.com/[Docker documentation].
+
+Built on existing Linux container technology (LXC), the open source project that is named Docker defined templates for how to package software into standardized units, called containers, that include all of the elements that an app needs to run. ROSA uses `cri-o` as the container runtime to deploy containers from Docker container images into your cluster.
+
+Containers are agile, small, and portable.
+
+Image:: A container image is the base for every container that you want to run. Container images are built from a Dockerfile, a text file that defines how to build the image and which build artifacts to include in it, such as the app, the app's configuration, and its dependencies. Images are always built from other images, making them quick to configure.
+
+Registry:: An image registry is a place to store, retrieve, and share container images. Images that are stored in a registry can either be publicly available (public registry) or accessible by a small group of users (private registry). ROSA offers public images that you can use to create your first containerized app. For enterprise applications, use a private registry to protect your images from being used by unauthorized users.
+
+Container:: Every container is created from an image. A container is a packaged app with all of its dependencies that can be moved between environments and run without changes. Unlike virtual machines, containers do not virtualize a device, its operating system, and the underlying hardware. Only the app code, run time, system tools, libraries, and settings are packaged inside the container. Containers run as isolated processes on Ubuntu compute hosts and share the host operating system and its hardware resources. This approach makes a container more lightweight, portable, and efficient than a virtual machine.
```

Comment:
Kind of weird that there's a "Container" definition under the "Container" concept, which already had a definition up above in the concept's intro, too. Could the info be combined into the intro definition?

Reply:
Done.